include/wx/charts/wxchartgrid.h

```diff
@@ -128,6 +128,9 @@ class wxChartGrid : public wxChartElement
     void Shift(double dx,double dy);
     void ChangeCorners(wxDouble minX,wxDouble maxX,
         wxDouble minY,wxDouble maxY);
+    void ChangeCorners(wxDouble min,wxDouble max,bool is_X = true);
+    void ChangeLabels(const wxVector<wxString> &labels,
```

Comment:
Modify this to void ChangeLabels(const std::string& axisId, const wxVector<wxString> &label)